src/custom/pages/Account/index.tsx

```diff
@@ -63,7 +63,7 @@ import { SwapVCowStatus } from 'state/cowToken/actions'
 import LockedGnoVesting from './LockedGnoVesting'
 import useBlockNumber from 'lib/hooks/useBlockNumber'
 import usePrevious from 'hooks/usePrevious'
-import { useCowFromLockedGnoBalances } from 'pages/Profile/LockedGnoVesting/hooks'
+import { useCowFromLockedGnoBalances } from '@src/custom/pages/Account/LockedGnoVesting/hooks'
```

Comment:
Should the imports here (line 27 and 66) be from `@srcpages/custom/pages/Account` rather than `pages/Account`?

Reply:
Yes you're right. I believe this has been fixed in `improve-header-12` (last branch).